src/App.js

```diff
@@ -3,22 +3,7 @@ import './App.css';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
You cannot remove everything in the return statement. If you run npm start you should get an error like this. Return at least one div.